### PR TITLE
Multiple flyer preparation

### PR DESCRIPTION
**Pull request by AbbyGi**

### CODE REVIEW

File: sirepo_bluesky/sirepo_flyer.py [hunk 1]
```diff
@@ -318,17 +332,26 @@ def collect(self):
...
-                    f'{self.name}_parameters': self.params_to_change[i],
+                    # f'{self.name}_parameters': self.params_to_change[i],
                     }
+            for inputs in self.params_to_change:
+                for key, params in inputs.items():
+                    elem_name.append(key)
+                    curr_param.append(list(params.keys()))
+            for ii in range(len(elem_name)):
+                for jj in range(len(curr_param[ii])):
+                    data[f'{self.name}_{elem_name[ii]}_{curr_param[ii][jj]}'] =\
```
Comment (mrakitin) [inline sirepo_bluesky/sirepo_flyer.py]:
```suggestion
                    data[f'{self.name}_{elem_name[ii]}_{curr_param[ii][jj]}'] = \
```

**Comment by mrakitin** [comment]
A lot of tests failed, see e.g. https://travis-ci.com/github/NSLS-II/sirepo-bluesky/jobs/298929950:
```
============================= test session starts ==============================
platform linux -- Python 3.6.7, pytest-4.3.1, py-1.7.0, pluggy-0.8.0 -- /home/travis/virtualenv/python3.6.7/bin/python
cachedir: .pytest_cache
rootdir: /home/travis/build/NSLS-II/sirepo-bluesky, inifile:
collected 8 items / 4 deselected / 4 selected                                  
sirepo_bluesky/tests/test_sirepo_det.py::test_sirepo_detector_vcr FAILED [ 25%]
sirepo_bluesky/tests/test_sirepo_det.py::test_sirepo_det_grid_scan_vcr FAILED [ 50%]
sirepo_bluesky/tests/test_sirepo_flyer.py::test_smoke_sirepo_vcr PASSED  [ 75%]
sirepo_bluesky/tests/test_sirepo_flyer.py::test_sirepo_flyer_vcr FAILED  [100%]
=================================== FAILURES ===================================
```
Was something wrong with the cassettes?

**Comment by AbbyGi** [comment]
I'll try remaking the cassettes and see if that helps at all.

**Comment by mrakitin** [comment]
I think it's something that the recently released databroker v1.0 caused:
```
>       self._resource_id = self.reg.insert_resource('srw', srw_file, {'ndim': ndim})
E       AttributeError: 'Registry' object has no attribute 'insert_resource'
```

**Comment by mrakitin** [comment]
The test failures will be addressed via #7. This PR's changes look good, and they are needed to perform simulations.

**Pull request merged**